proposals/0001-simd-process.md

```diff
@@ -130,7 +136,8 @@ is descriptive)
 - Fill in the proposal. Put care into the details: proposals that do not
 present convincing motivation, demonstrate lack of understanding of the
 design's impact, or are disingenuous about the drawbacks or alternatives tend
-to be poorly received.
+to be poorly received. Low quality proposals with limited engagement will be 
```

Comment:
How would you define `limited engagement`? It may be easier to define a length of time such as what we had with `stagnant`.

Reply:
I just got rid of the "limited engagement" text.